meta/runtime.yml

```diff
@@ -15,6 +15,7 @@ action_groups:
   proxmox:
     - proxmox
     - proxmox_disk
+    - proxmox_backup
```

Comment:
`b` comes before `d` :)
```suggestion
    - proxmox_backup
    - proxmox_disk
```

Reply:
Fix in 923fbd76014272b458eba79cdc257d9910718e23